src/components/MySongsTable/MySongsTable.js

```diff
@@ -68,13 +56,17 @@ export class MySongsTableComponent {
   }
 
   mount() {
-    this.mountPoint.innerHTML = this.render();
-    this.querySelectors();
-    this.initMaterial();
-    this.mountChildren();
+    MusicService.getAlbumSongs("album1").then(songs => {
```

Comment:
no. First should should render empty table. And then fetch data. And then render table with data

Reply:
If I correctly understand the task is just to come up with approach for handling playing song from a list. The pages with these tables are developed by other guys. Does it make sense to fix this logic here?

Reply:
yes. Your task is to implement playing from my songs list.